samples/getting-started/simulation/LargeSimulation.ipynb

```diff
@@ -0,0 +1,2459 @@
+    "open Microsoft.Quantum.Diagnostics;   // Contains subroutines that help ensure correctness of Q# programs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "affiliated-value",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[\"PrepareAndMeasureGhzState\"]",
+      "text/html": [
+       "<ul><li>PrepareAndMeasureGhzState</li></ul>"
+      ],
+      "text/plain": [
+       "PrepareAndMeasureGhzState"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "/// # Summary\n",
+    "/// Prepares, measures, and resets the GHZ states.\n",
+    "///\n",
+    "/// # Input\n",
+    "/// ## nQubits\n",
+    "/// Number of qubits to allocate (must be multiple of the nClusters input).\n",
+    "/// \n",
+    "/// ## nClusters\n",
+    "/// Number of clusters to break-up qubits into.\n",
+    "operation PrepareAndMeasureGhzState(nQubits : Int, nClusters : Int) : Unit {\n",
+    "    Message(\"Let's run a big simulation!\");\n",
+    "\n",
+    "    Message($\"Allocating {nQubits} qubits...\");\n",
+    "    use qs = Qubit[nQubits];                        // Allocate the qubits.\n",
+    "    Message(\"Finished allocating!\");\n",
+    "\n",
+    "    let clusterSize = nQubits / nClusters;          // Number of qubits in every cluster.\n",
+    "    Message($\"Creating {nClusters} clusters of GHZ states between {clusterSize} qubits...\");\n",
+    "    for cluster in Chunks(clusterSize, qs) {        // Iterate through the clusters.\n",
+    "        H(Head(cluster));                           // Apply H gate to the first-most qubit in the cluster.\n",
+    "        ApplyCNOTChain(cluster);                    // Apply CNOT gate to the remaining qubits in the cluster.\n",
+    "                                                    // After this, all qubits in the cluster will have the same state.\n",
+    "    }\n",
+    "    Message(\"Finished GHZ clusters!\");\n",
+    "\n",
+    "    if nQubits < 15 {                               // For small states\n",
+    "        DumpMachine();                              // show the histogram.\n",
+    "    }\n",
+    "\n",
+    "    Message(\"Collapse and show final state:\");\n",
+    "    let results = ForEach(M, qs);                   // Measure the qubits.\n",
+    "    Message($\"{results}\");\n",
+    "\n",
+    "    Message(\"Cleaning up...\");\n",
+    "    ResetAll(qs);\n",
+    "\n",
+    "    Message(\"Done!\");\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "flying-business",
+   "metadata": {},
+   "source": [
+    "We'll make the small states displayed conveniently on a histogram.  \n",
+    "For more information about `dump.basisStateLabelingConvention`, please see [Configuration settings](https://docs.microsoft.com/en-us/qsharp/api/iqsharp-magic/config#configuration-settings)."
```

Comment:
```suggestion
    "For more information about `dump.basisStateLabelingConvention`, please see [Configuration settings](https://docs.microsoft.com/qsharp/api/iqsharp-magic/config#configuration-settings)."
```

Reply:
Fixing. I apologize for having you to report this multiple times.

Reply:
No worries at all, it's easy to miss!